CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+2.1.4 (in progress)
+===================
+
+New Features
+------------
+ - Removed workaround for this Chrome [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=774303).
+  Now, we no longer suppress the RTCPeerConnection's native `RTCTrackEvent`.
```

Comment:
Since this is a new feature, we must increment the minor version and we should probably branch to support-2.2.x, too.

Nitpicks:
* Please insert a newline before the bullet point.
* Please indent one space.
* I recommend we do not put RTCTrackEvent—or any class names—in back ticks.